swap_meet/item.py

```diff
@@ -1,2 +1,26 @@
+
+
 class Item:
-    pass
+
+    def __init__(self, category='', condition=None):
+        self.category = category
+        self.condition = condition
+        
+    def __str__(self):
+        return 'Hello World!'
+
+    def condition_description(self):
+        if self.condition == 0:
```

Comment:
The condition is specified as a float. So what would happen here if the condition value were 3.5? Is the behavior that would happen what you would expect? How might you adjust this logic?